quinn-proto/src/connection.rs

```diff
@@ -3243,6 +3260,22 @@ struct SentPacket {
 /// Ensures we can always fit all our ACKs in a single minimum-MTU packet with room to spare
 const MAX_ACK_BLOCKS: usize = 64;
 
+/// Events to be sent to the Connection
+pub enum ConnectionEvent {
+    NewIdentifiers([Option<ConnectionId>; LOC_CID_COUNT]),
+}
+
+/// Events to be sent to the Endpoint
+#[derive(Clone, Debug)]
+pub enum EndpointEvent {
+    ClientReady,
+    Incoming,
+    Closed {
+        loc_cids: Vec<ConnectionId>,
```

Comment:
Could this list be maintained inside `endpoint::ConnectionMeta` rather than being sent back and forth?

Reply:
Good idea, see b82110cb3a33a9db7d83f0512ec4f0fc278dd961.